test/integrations/destinations/af/processor/data.ts

```diff
@@ -1716,6 +1716,98 @@ export const existingTestCases = [
       },
     },
   },
+  {
+    name: 'af',
+    description: 'Place selected properties at root track call with af data',
+    id: 'selectedPropsAtRoot',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: {
+              type: 'track',
+              event: 'Order Completed',
+              sentAt: '2020-08-14T05:30:30.118Z',
+              context: commonContextWithExternalId,
+              messageId: '7208bbb6-2c4e-45bb-bf5b-ad426f3593e9',
+              timestamp: '2020-08-14T05:30:30.118Z',
+              properties: {
+                ...commonPropertiesWithProduct,
+                prop1: 'value1',
+                prop2: 'value2',
+                Prop3: 'value3',
+              },
+              anonymousId: '50be5c78-6c3f-4b60-be84-97805a316fb1',
+              integrations: { AF: { af_uid: 'afUid' } },
+            },
+            destination: {
+              Config: {
+                devKey: 'abcde',
+                androidAppId: 'com.rudderlabs.javascript',
+                groupTypeTrait: 'email',
+                groupValueTrait: 'age',
+                trackProductsOnce: false,
+                trackRevenuePerProduct: false,
+                addPropertiesAtRoot: false,
+                listOfProps: [
+                  {
+                    property: 'prop1',
+                  },
+                  {
+                    property: 'prop2',
+                  },
+                  {
```

Comment:
remove prop3 from here to cover 2 scenarios of being at root vs. properties

Reply:
This is also intensional to check the case sensitivity.

Reply:
In that case, Prop3 should not be at root level ?